tests/dsp/pdr.cpp

```diff
@@ -2238,12 +2238,20 @@ TEST(EntityAssociationPDR, testAddContainedEntityNew)
 
 TEST(EntityAssociationPDR, testRemoveContainedEntity)
```

Comment:
Does the testcase runs fine without adding the new child? If yes can we keep the existing case as it is add new cases on top

Reply:
yes